# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/url-classifier/moz.build
# Commit: 8d3727e19ade
# Full Hash: 8d3727e19ade1f2373c3087b709ccb7a42249b6c
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2022-07-19 21:35:33
# Regressor Bug: 1773701
# File Overlap Count: 1
# Description:
#   Bug 1773701 - Part 2: Implement the email tracking data collection feature. r=dimi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D151522
# ==============================================================================

diff -r fab41f0c857f -r 8d3727e19ade netwerk/url-classifier/moz.build
--- a/netwerk/url-classifier/moz.build	Tue Jul 19 14:46:07 2022 +0000
+++ b/netwerk/url-classifier/moz.build	Tue Jul 19 14:46:08 2022 +0000
@@ -36,6 +36,7 @@
     "UrlClassifierFeatureCryptominingAnnotation.cpp",
     "UrlClassifierFeatureCryptominingProtection.cpp",
     "UrlClassifierFeatureCustomTables.cpp",
+    "UrlClassifierFeatureEmailTrackingDataCollection.cpp",
     "UrlClassifierFeatureEmailTrackingProtection.cpp",
     "UrlClassifierFeatureFactory.cpp",
     "UrlClassifierFeatureFingerprintingAnnotation.cpp",
